x-pack/plugins/monitoring/public/components/kibana/instances/instances.js

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { PureComponent } from 'react';
+import { EuiPage, EuiPageBody, EuiPageContent, EuiPanel, EuiSpacer, EuiLink } from '@elastic/eui';
+import { capitalize } from 'lodash';
+import { ClusterStatus } from '../cluster_status';
+import { EuiMonitoringTable } from '../../table';
+import { KibanaStatusIcon } from '../status_icon';
+import { formatMetric, formatNumber } from '../../../lib/format_number';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+const getColumns = (kbnUrl, scope) => {
```

Comment:
I noticed none of these columns have `sortable` which will be set to `true` automagically by `EuiMonitoringTable`, but in some cases it might require a handler ((row) => row.sortOnThisPropInstead) to sort on a different property.

This is just something to account for, but might not require any change here